client-cli/src/command/transaction_command.rs

```diff
@@ -26,6 +28,7 @@ use crate::{ask_seckey, coin_from_str};
 pub enum TransactionType {
     Transfer,
     Deposit,
+    DepositAmount,
```

Comment:
some comment?
explaining deposit, deposit amount is different

Reply:
yes, that would be good -- especially, it's not really a different transaction type... it's two transactions in a sequence (Transfer => Deposit)

Reply:
Ok